### DISCUSSION TITLE
DAMMIN having worse fits than GNOM

### DISCUSSION THREAD

**Discussion by franke-biosaxs** [biosaxs-com/atsas-community#963] [Ab initio shape determination]
On 2013-03-18 23:06:59, user ankimble wrote:

Hi.  I've been trying to use DAMMIN to fit my GNOM output data and the fits have gotten worse instead of fitting better.  It was suggested that there might be a problem with my GNOM output (v 4.6). I have been using the online DAMMIN program.  I have attached the the DAMMIN output as a zip file, and it also contains the GNOM output.

Attachments:

  - [The zip file from DAMMIN](https://github.com/biosaxs-com/atsas-community/blob/saxier-org/forum/attachments/26577_979c57c4c8074564812a3f97c161b643.zip)

**Comment by franke-biosaxs** [comment]
On 2013-03-18 23:43:20, user franke wrote:


ankimble wrote
> Hi.  I've been trying to use DAMMIN to fit my GNOM output data and the fits have gotten worse instead of fitting better.

I might be a bit slow tonight, but could you maybe explain what you expected? In more words?

Looking at the gnom file: job type 6, shell case is correct? Your parameters are correct? What exactly is it that you want to model with DAMMIN? About the model: the beads are too tightly packed, that usually indicates an issue with the input units, i.e. Angstrom vs nanometer.

In short: verify your inputs ...

HTH

Daniel

**Comment by franke-biosaxs** [comment]
On 2013-03-25 19:43:24, user ankimble wrote:

I wanted to use DAMMIN to get a bead model for the protein structure.

The online ATSAS DAMMIN only asks for the GNOM output file, Symmetry, Anisometry and Mode. Is there any way that the inputs might change between inputting in GNOM and the GNOM output that is used for DAMMIN?

**Comment by franke-biosaxs** [comment]
On 2013-03-26 10:29:31, user SaxsMax wrote:

Would you mind posting also the original data, i.e. not processed by Gnom?
Where was it measured? At a synchrotron or on a lab source?
In the latter case the data is probably smeared.
Were several concentrations measured, is there any concentration effect?

Also, is the sample just a protein or a protein in complex with lipids or sugars?
Dammin/f are only applicable for quasi-homogeneous particles,
i.e. lipoprotein would not be reasonably reconstructed.

**Comment by franke-biosaxs** [comment]
On 2013-03-27 18:38:56, user ankimble wrote:

The original data was collected at Argonne APS 12-ID-B. It has all of the buffer components subtracted out.  The only other component that was in the buffer that may still be interacting with the protein is a detergent, dodecyl thiomaltopyranoside, but it is 0.013g/L which is below the cmc.

Attachments:

  - [26577_f5272463f079a1abbdfb3da660eae8ec.txt](https://github.com/biosaxs-com/atsas-community/blob/saxier-org/forum/attachments/26577_f5272463f079a1abbdfb3da660eae8ec.txt)

**Comment by franke-biosaxs** [comment]
On 2013-03-27 19:35:56, user SaxsMax wrote:

The presence of detergent is to my mind the source of the problems with ab initio modelling from this data.

Also, the experimental errors are missing (zeroed) which affects the Gnom analysis.
